### DISCUSSION TITLE
edit text

### DISCUSSION THREAD

**Discussion by ivanbishop** [Stirling-Tools/Stirling-PDF#871] [Q&A]
Can I enter text into empty fields of an Adobe generated PDF with Stirling, eg. form fill?
thanks

**Comment by sbplat** [comment]
Filling out forms should be possible directly through most web browsers. Do you want to accomplish this via an API?

**Reply by ryan-morris** [reply]
That's the ability I was looking for. We have a lot of templates with fillable fields. Would like to send in a PDF and field name/values to set the values and get new pdf back out. The more complicated bit is we do make use of image fields. Above and beyond for filling in fields would be ability to disable the fields in the returned pdf.

**Reply by lingzhou125** [reply]
Hi -- edit text would be really helpful for us as well. we have a workflow where we currently need someone to download, edit, then re-upload a pdf. it would be nice if it can all be done in app!
